docs/azure-monitor-logging.md

```diff
@@ -0,0 +1,81 @@
+# Azure Monitor tagging
+
+All telemetry sent to Azure Monitor, whether traces, metrics, or spans, should follow this guide.
+
+## Overloading of default dimensions
+
+A few built-in Azure Monitor fields are overloaded:
+
+| field | overload |
+| - | - |
+| `cloud_RoleName` | Either `device` or `service` |
+| `cloud_RoleInstance` | `runId` for the process producing telemetry. |
+
+## customDimensions
+
+```json
+  {
+    "sdkLanguageVersion": "3.8.2",
+    "sdkLanguage": "python",
+    "sdkVersion": "2.2.3",
+    "lineNumber": "291",
+    "sessionId": "a8b42042-6a08-4bcd-85c3-3d96d25ccd64",
+    "fileName": "service.py",
+    "deviceId": "bertk_test_device_4",
+    "process": "MainProcess",
+    "module": "service",
```

Comment:
What is module?

Reply:
python specific.  moving lower in the doc